Bottom: 5de0db34c98e15b3bb7cb0fd89a595b96b8c8dd7
Top:    b303cd3620999efa501388c7a24b4b5df5363331
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2009-10-28 17:41:00 +0000

Not sure what this patch is for ;)


---

diff --git a/src/draw.c b/src/draw.c
index 59e5e31..6089eba 100644
--- a/src/draw.c
+++ b/src/draw.c
@@ -530,7 +530,7 @@ DrawEverything (BoxTypePtr drawn_area)
 	r_search (PCB->Data->element_tree, drawn_area, NULL, EMark_callback,
 		  NULL);
       /* Draw rat lines on top */
-      if (PCB->RatOn)
+      if (PCB->RatOn && gui->set_layer ("rats", SL (RATS, 0), 0))
 	DrawRats(drawn_area);
     }
